cmd/plugin_manager.go

```diff
@@ -117,3 +117,26 @@ func createPluginManagerLog() hclog.Logger {
 	log.SetFlags(0)
 	return logger
 }
+
+func cleanupOldTmpDirs() {
+	const tmpDirAgeThreshold = 24 * time.Hour
+	tmpDirPattern := filepath.Join(filepaths.EnsurePluginDir(), "tmp-*")
+	tmpDirs, err := filepath.Glob(tmpDirPattern)
```

Comment:
Yes you are right, `filepath.Glob` would do the job here. 

But we use our [turbot/go-kit](https://pkg.go.dev/github.com/turbot/go-kit@v0.7.0) func [ListFiles](https://github.com/turbot/go-kit/blob/232b68f04b52d2e2409568cf40ee8fa7a4c08c08/files/list.go#L43) in cases like these to list files and directories.

Here's an example usage:
https://github.com/turbot/steampipe/pull/3494/files#diff-61ef52fca4e6c3ae75d4641572fe075dbadc6beddcbce7a22a9fb9bd674f08caR96